src/Symfony/Component/PasswordHasher/Hasher/PasswordHasherFactory.php

```diff
@@ -143,6 +148,8 @@ private function getHasherConfigFromAlgorithm(array $config): array
 
             foreach ($frompasswordHashers as $name) {
                 if ($hasher = $this->passwordHashers[$name] ?? false) {
+                    $hasher = $this->createHasherUsingAdapter($name);
+
                     $hasher = $hasher instanceof PasswordHasherInterface ? $hasher : $this->createHasher($hasher, true);
```

Comment:
Do we still need that line?

Reply:
@derrabus it is the only useful line in the patch / the only line I add... the rest of the patch is just factorizing to avoir duplicate code

The useless part is the assignation in `if($hasher = $this->passwordHashers[$name] ?? false)` that I could replace with `if(isset($this->passwordHashers[$name]))`

Reply:
@derrabus I may have misunderstood which line you are talking about. I updated the code, can you please look again? Thanks!